src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,133 @@
+                        String startTime = command.getStartTime();
+                        String endTime = command.getEndTime();
+
+                        return new EventTask(description, false, date, startTime, endTime);
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        throw new InvalidTaskException("OOPS!!! The description, date, start time or "
+                                + "end time of an event cannot be empty.");
+                    }
+                }
+                default:
+                    throw new InvalidTaskException("OOPS!!! I'm sorry, but I don't what task is this. :-(");
+            }
+        } catch (InvalidDateException e) {
+            throw new InvalidTaskException("OOPS!!! Please enter a valid "
+                    + "date and time in the format dd-mm-yyyy.");
+        }
+        catch (InvalidTimeException e) {
+            throw new InvalidTaskException("OOPS!!! Please enter a valid "
+                    + "time in the format HHmm.");
+        }
+    }
+
+    /**
+     * Parse the user input and returns a LocalDate object.
+     * @param date The date string to be parsed.
+     * @return The LocalDate object created from the date string.
+     * @throws InvalidDateException If the date string is invalid.
+     */
+    public static LocalDate parseDate(String date) throws InvalidDateException {
+        try {
+            return LocalDate.parse(date, DateTimeFormatter.ofPattern("dd-MM-yyyy"));
+        } catch (Exception e) {
+            throw new InvalidDateException("Invalid date format. Please use the format dd-MM-yyyy.");
+        }
+    }
+
+    /**
+     * Parse the LocalDate object to a nice format string.
+     * @param date The LocalDate object to be parsed.
+     * @return The string representation of the LocalDate object.
+     */
+    public static String dateToString(LocalDate date) {
+        return date.format(DateTimeFormatter.ofPattern("dd MMM yyyy"));
+    }
+
+    /**
+     * Parse the LocalDate object to a storage format string.
+     * @param date The LocalDate object to be parsed.
+     * @return The string representation of the LocalDate object.
+     */
+    public static String dateToStorageString(LocalDate date) {
+        return date.format(DateTimeFormatter.ofPattern("dd-MM-yyyy"));
+    }
+
+    /**
+     * Parse the LocalTime object to a storage format string.
+     * @param time The LocalTime object to be parsed.
+     * @return The string representation of the LocalTime object.
+     */
+    public static String timeToStorageString(LocalTime time) {
+        return time.format(DateTimeFormatter.ofPattern("HHmm"));
+    }
+
+    /**
+     * Parse the LocalTime object to a nice format string.
+     * @param time The LocalTime object to be parsed.
+     * @return The string representation of the LocalTime object.
+     */
+    public static String timeToString(LocalTime time) {
```

Comment:
This ``timeToString`` method and the names of other 3 methods above seem not to be of a verb type, aren't they?